app/assets/javascripts/discourse/components/site-header.js

```diff
@@ -205,6 +205,30 @@ const SiteHeaderComponent = MountWidget.extend(Docking, PanEvents, {
     this.dispatch("search-autocomplete:after-complete", "search-term");
 
     this.appEvents.on("dom:clean", this, "_cleanDom");
+
+    // Allow first notification to be dismissed on a click anywhere
+    if (
+      !this.get("currentUser.read_first_notification") &&
+      !this.get("currentUser.enforcedSecondFactor")
+    ) {
+      document.addEventListener(
```

Comment:
There should be an equivalent removeEventListener in the willDestroyElement. 

(I realize in normal Discourse operation the header is never removed, but in an test environment you want to make sure to get this stuff right.)

Reply:
That's what I get for brain fogging my way through this, thanks for catching that - I was thinking that because it was a "once" event, it'd remove itself (in most cases), but I was forgetting about test case cleanup! Updated once more.